web-client/terraform/main/main.tf

```diff
@@ -168,8 +168,9 @@ resource "aws_route53_health_check" "public_ui_health_check" {
   port              = 443
   type              = "HTTPS"
   resource_path     = "/"
-  failure_threshold = "2"
+  failure_threshold = "3"
   request_interval  = "30"
+  regions           = [ "us-east-1", "us-west-1", "us-west-2" ] # Minimum of three regions required
```

Comment:
Why not continue using the default set of regions that AWS uses? They're global... Just curious.

Reply:
Thanks for the question! It’s very confusing, and I just learned this! By overriding these regions, we reduce health checking traffic by 62%. 

When configuring Route53 health checks, you’re actually configuring health check workers _per AWS datacenter_. I assume this is so you can regionally react based on latency and scale per-data center. 

The result is that when you use the default regions, you’re actually configuring workers in **8 data centers** which results in approximately 15x higher traffic than configured here. I measured this observed traffic at 1,779 requests per hour per environment.

By overriding to custom regions here, we configure workers in **3 data centers** instead. This results in observed traffic of 671 requests per hour per environment. 

You can follow along with my learning on [the original issue](https://github.com/ustaxcourt/ef-cms/issues/712#issuecomment-760258118), including the graphs used to calculate the observed traffic.